### PR TITLE
Docs: add generic endpoints to the EDC postman collections

### PR DESCRIPTION
**Pull request by nicoprow**
## Description

Added the generic endpoints to the existing EDC Postman collections analogous to the other endpoints.

Solves issue #509 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/edc/EDC_BPDM_Setup.postman_collection.json [hunk 1]
```diff
@@ -1,1391 +1,1524 @@
...
-    },
-    {
-      "key": "ASSET_POST_CHANGELOG_POOL_SEARCH",
-      "value": "POST_CHANGELOG_POOL_SEARCH",
-      "type": "string"
-    }
-  ]
+	"info": {
+		"_postman_id": "aab9dc22-2777-4429-8e0e-e72311c54d6b",
+		"name": "EDC BPDM Setup Template",
```
Comment (SujitMBRDI) [inline docs/edc/EDC_BPDM_Setup.postman_collection.json:4]: Can we change the name, EDC BPDM Setup Template -> EDC BPDM Setup

File: docs/edc/EDC_BPDM_Usage.postman_collection.json [hunk 2]
```diff
@@ -487,32 +6335,52 @@
     }
   ],
   "variable": [
+    {
+      "key": "SHARING_MEMBER_EDC_BASE_URL",
```
Comment (SujitMBRDI) [inline docs/edc/EDC_BPDM_Usage.postman_collection.json:6339]: This is really great, depending on environment we can adapt the sharing member EDC and gate EDC base urls

File: docs/edc/EDC_BPDM_Usage.postman_collection.json [hunk 3]
```diff
@@ -1,454 +1,6302 @@
 {
   "info": {
-    "_postman_id": "67fa306a-a4c1-4d70-80bb-370491d8d0a9",
-    "name": "EDC BPDM Usage",
+    "_postman_id": "478bfefa-0ab2-4299-993e-1800562484f5",
+    "name": "EDC-BPDM Automated Test Template",
```
Comment (SujitMBRDI) [inline docs/edc/EDC_BPDM_Usage.postman_collection.json:4]: All looks good, just one comment as above for name value.
Also, I have imported collection and tried testing the same but OfferID for Generic endpoints are not being generated so couldn't go into further the details but test scripts looks good to me.

We have automation folder in repo with file /main/docs/postman/automated-test/EDC-BPDM%20Automated%20Test.postman_collection.json either we need to remove that and keep this or vice versa  with updated changes for Generic endpoints will also be fine .

File: docs/edc/EDC_BPDM_Setup.postman_collection.json (hunk 1 above)
Comment (nicoprow) [reply docs/edc/EDC_BPDM_Setup.postman_collection.json]: Changed the name back to the original

File: docs/edc/EDC_BPDM_Usage.postman_collection.json (hunk 3 above)
Comment (nicoprow) [reply docs/edc/EDC_BPDM_Usage.postman_collection.json:4]: I tested it again and fixed some issue with the setup. Now it should work on our EDC

**Pull request merged**